apps/teca_temporal_reduction.in

```diff
@@ -41,10 +32,41 @@ parser.add_argument('--operator', type=str, default='average',
                     help='reduction operator to use. One of minimum, '
                          'maximum, or average (average)')
 
+parser.add_argument('--point_arrays', nargs='+', required=True,
+                    help='list of point centered arrays to process.')
+
+parser.add_argument('--fill_value', type=float, default=None,
+                    help='a value that identifies missing or invalid data.')
+
+parser.add_argument('--use_fill_value', type=int, default=0,
```

Comment:
The wording and typing of the argument here are ambiguous: it's not immediately clear whether this is meant to be a boolean flag or whether this is intended as a way to manually set the value of the fill value.

I see from the revised code that this is used as a boolean flag. So I suggest to either (1) use a bool type for the variable in argparse, or (2) modify the documentation to make clear that any non-zero value will result in the fill value being used.

Also, I notice that the default value here is 0, whereas the default value in `alg/teca_temporal_reduction.py` is 1.  I think a default value of 1 is preferable everywhere; in general, most climate data analysis tools honor the fill value by default.

Reply:
done